mysql-to-kv-delta/mysql-to-kv-delta.py

```diff
@@ -0,0 +1,70 @@
+
+
+      
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import datetime
+
+import pandas as pd
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import v3io_frames as v3f
+
+
+# MYSQL db variables
+SQL_QUERY = os.environ['SQL_QUERY']
+SQL_HOST = os.environ['SQL_HOST']
+SQL_PORT = os.environ['SQL_PORT']
+SQL_USER = os.environ['SQL_USER']
+SQL_PWD = os.getenv('SQL_PWD', "")
+SQL_DB_NAME = os.environ['SQL_DB_NAME']
+
+# MYSQL query variables
+DELTA_INTERVAL_MINUTE = os.environ['DELTA_INTERVAL_MINUTE']
+MODIFIED_DATETIME_COL = os.environ['MODIFIED_DATETIME_COL']
+CREATED_DATETIME_COL = os.environ['CREATED_DATETIME_COL']
+
+
+def handler(context, event):
+    datetime_query = str(datetime.datetime.now() - datetime.timedelta(seconds=int(DELTA_INTERVAL_MINUTE)*60))
+    sql_query_diff = SQL_QUERY + " where ("+CREATED_DATETIME_COL+">='"+str(datetime_query)+"' AND "+MODIFIED_DATETIME_COL+" IS NULL) OR ("+MODIFIED_DATETIME_COL+">='"+str(datetime_query)+"')"
+    # for debugging the generated sql query
+    context.logger.debug(sql_query_diff)
+    df = pd.read_sql(sql_query_diff, context.dbconn.connection())
+     # for debugging no of upserts
+    context.logger.debug("no of delta rows :: {} ".format(df.shape[0]))
```

Comment:
Capitalize first word of log message, and follow logging format: (I'm not sure what the meaning of this shape here is, change it to whatever you think is appropriate. I'm just showing example of our logging structure)
```
context.logger.debug('Number of delta rows', shape=df.shape[0])
```